### PR TITLE
TSE Review

### PR DESCRIPTION
**Pull request by Victorgonbu**
Added all the elements and style for the homepage, applying the flexbox, grid elements, and the responsive design.

# Navigation Bars
- Defined the position
- Included Buttons and Logo.

# Section and Articles
- Included the elements
- Added the paragraphs
- Embedded images
- Position the section and its elements
- Style the text and background color

# Footer
- Styled and positioned the footer lists
- Positioned all the list with its links

# Media query 
- added responsive design for 3 breakpoint (375px, 765px and 1024+px )

- We also made the recommended validations with Linter and W3 Validator

### CODE REVIEW

File: assets/css/media.css [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    justify-content: center;
+  }
+
+  footer div.legal span {
+    display: inline-block;
+    margin: 5px;
+  }
+}
+
+/* Breakpoint w:1024px */
```
Comment (voscarmv) [inline assets/css/media.css]: Please remove **all** comments from your CSS. This will make your code look cleaner ✨

File: assets/css/media.css [hunk 2]
```diff
@@ -0,0 +1,343 @@
+@media (min-width: 765px) {
```
Comment (voscarmv) [inline assets/css/media.css]: The `min-width` should be set to `768px`

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Excellent project so far team! 💪 👨‍💻 
## However, some changes are required before you may continue... 🧰 🔨 🔧 
### In the *Latest Funding section*
- ### For medium display, the page must align 3 articles per row.
### 👇  👇  👇 Your page 👇  👇  👇 ❌ 
image
### 👇  👇  👇  Original page:  👇  👇  👇 ✔️ 
image

### Also, please read the comments I've left for you on your code for additional changes! ⚙️ ⚙️ 

### Questions or comments? Feel free to contact me! www.twitter.com/voscarmv www.linkedin.com/in/oscar-mier  🧑‍💻 💬 🧑‍💻  

# Happy Coding!

File: assets/css/media.css (hunk 1 above)
Comment (Victorgonbu) [reply assets/css/media.css]: Thank for the suggestion we removed all the comments

File: assets/css/media.css (hunk 2 above)
Comment (Victorgonbu) [reply assets/css/media.css]: Thanks for the suggestion we set the min-width to 768px

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @Shaqri @wrakc** 
Amazing job :clap: :clap: you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with your master branch

Keep going like this,

Oscar Andrés Russi Porras

**Pull request merged**